tests/test_units.py

```diff
@@ -82,28 +82,30 @@ def test_get_dimension(self, session):
         assert len(dimension.units) > 0, \
             "Could not get the dimension units for Dimension ID 1."
 
-        with pytest.raises(ResourceNotFoundError):
+        with pytest.raises(HydraError):
             # It must raise an exception
-            dimension = hb.get_dimension(None, do_accept_dimension_id_none=False)
+            dimension = client.get_dimension(None, do_accept_dimension_id_none=False)
 
-        dimension = hb.get_dimension(None, do_accept_dimension_id_none=True)
+        dimension = client.get_dimension(None, do_accept_dimension_id_none=True)
         assert dimension is not None, \
             "Could not get a dimension by None."
         assert dimension.id is None, \
             "Could not get a dimension by None."
 
 
-    def test_get_dimensions(self, session):
-        # Returns an array of dimensions. Every item contains the dimension data plus the units list
-        dimension_list = hb.get_dimensions()
+    def test_get_dimensions(self, client):
+        # Returns an array of dimensions. Every item contains the
+        # dimension data plus the units list
+        dimension_list = client.get_dimensions()
 
         assert dimension_list is not None and len(dimension_list) != 0, \
```

Comment:
As the `len` is intended to demonstrate that there is at least one dimension in the list, should we also test for the presence of an `id` attr (i.e. `dimension.id`) to confirm that it is indeed a dimension?